app/components/form.tsx

```diff
@@ -78,6 +77,39 @@ import { valueComparator } from "@/utils/sorting-values";
 import useEvent from "@/utils/use-event";
 import "@mdxeditor/editor/style.css";
 
+// @ts-ignore
+const useFormControlStyles = makeStyles((theme: Theme) => ({
+  sm: {
+    ...theme.typography.caption,
+  },
+  md: {
+    ...theme.typography.body3,
+  },
+  lg: {
+    ...theme.typography.h5,
+  },
+}));
```

Comment:
Wouldn't it be easier to change the typography variant below ? 

```
const sizeToVariant = {  sm: 'caption', .... }

.... below ...

<MUIFormControlLabel
      {...rest}
      componentsProps={{
        typography: {
          variant: sizeToVariant[size],
          as='label'
        },
      }}
    />
```

Reply:
Yes, that's indeed easier, thanks!